### PR TITLE
Chart: Default airflow version to 2.7.0

### PR DESCRIPTION
**Pull request by ephraimbuddy**

### CODE REVIEW

**Review by hussein-awala — APPROVED** [review]

**Review by jedcunningham — APPROVED** [review]

File: chart/Chart.yaml [hunk 1]
```diff
@@ -47,23 +47,23 @@ annotations:
...
+      url: https://airflow.apache.org/docs/apache-airflow/2.7.0/_images/dags.png
     - title: Datasets View
-      url: https://airflow.apache.org/docs/apache-airflow/2.6.3/_images/datasets.png
+      url: https://airflow.apache.org/docs/apache-airflow/2.7.0/_images/datasets.png
     - title: Grid View
-      url: https://airflow.apache.org/docs/apache-airflow/2.6.3/_images/grid.png
+      url: https://airflow.apache.org/docs/apache-airflow/2.7.0/_images/grid.png
     - title: Graph View
-      url: https://airflow.apache.org/docs/apache-airflow/2.6.3/_images/graph.png
+      url: https://airflow.apache.org/docs/apache-airflow/2.7.0/_images/graph.png
```
Comment (jedcunningham) [inline chart/Chart.yaml:56]: I checked... the new graph and gantt images have the same filename.

File: chart/Chart.yaml
Comment (pankajkoti) [inline chart/Chart.yaml:1]: Do we also need to update on L118 to update the description? It still says 2.6.3. 
I mean similar to what we did here https://github.com/apache/airflow/pull/32517/files#diff-462f538e090c491e7d916fd38843be03a198e0ed4f9c266b5704ef7af3244948R118 ?

File: chart/newsfragments/32517.significant.rst [hunk 2]
```diff
@@ -1,3 +1,3 @@
-Default Airflow image is updated to ``2.6.3``
+Default Airflow image is updated to ``2.7.0``
 
-The default Airflow image that is used with the Chart is now ``2.6.3``, previously it was ``2.6.2``.
+The default Airflow image that is used with the Chart is now ``2.7.0``, previously it was ``2.6.2``.
```
Comment (pankajkoti) [inline chart/newsfragments/32517.significant.rst:3]:
```suggestion
The default Airflow image that is used with the Chart is now ``2.7.0``, previously it was ``2.6.3``.
```

File: chart/Chart.yaml (hunk 1 above)
Comment (ephraimbuddy) [reply chart/Chart.yaml:56]: I don't fully follow. Both resolves to different filename(for the links here)

File: chart/newsfragments/32517.significant.rst (hunk 2 above)
Comment (ephraimbuddy) [reply chart/newsfragments/32517.significant.rst:3]: No. The previous release of the chart is still in 2.6.2 cc @jedcunningham

File: chart/Chart.yaml
Comment (ephraimbuddy) [reply chart/Chart.yaml:1]: So, I usually do search and replace on this file but today I had to look at it very well today and it seems I was wrong previously but I will like to hear what @jedcunningham thinks about it

File: chart/Chart.yaml (hunk 1 above)
Comment (jedcunningham) [reply chart/Chart.yaml:56]: I just mean, generally, we can't always just update the version in the url. Both graph and gantt were refactored in 2.7, so there was a possibility the new screenshot had a different filename. But they were updated this time around.

This was meant for future reviewers, just wanted to note that I already checked that these 1) exist and 2) are the right images.

File: chart/Chart.yaml
Comment (jedcunningham) [reply chart/Chart.yaml:1]: Yeah, that shouldn't have been updated last time around :)

Comment (jedcunningham) [reply chart/Chart.yaml:1]: It's not a huge deal though - that section gets replaced during the chart release process anyways so it'll have the right pr/description when we go to do the next release.

So no need to "fix" it here either.

**Pull request merged**